src/components/Audiences/index.js

```diff
@@ -18,13 +18,27 @@ const createAudiences = ({ logger }) => {
     fireReferrerHideableImage,
     logger
   });
+
   const processDestinationsFromResponse = ({ response }) => {
     const destinations = response.getPayloadsByType("activation:push");
     return processDestinations(destinations);
   };
+
+  const retrievePullDestinations = ({ response }) => {
+    return Promise.resolve({
+      destinations: response.getPayloadsByType("activation:pull")
+    });
+  };
+
+  const processResponse = ({ response }) => {
+    return processDestinationsFromResponse({ response }).then(() =>
+      retrievePullDestinations({ response })
+    );
+  };
+
   return {
     lifecycle: {
-      onResponse: processDestinationsFromResponse
+      onResponse: processResponse
     },
     commands: {}
   };
```

Comment:
I'm starting to think we should rename this Component to `Activation` instead of `Audiences` since it's broader.

Reply:
I will create a ticket for this